apis/apiextensions/v1beta1/zz_generated.composition_environment.go

```diff
@@ -1,368 +0,0 @@
-/*
-Copyright 2022 The Crossplane Authors.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-	http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-// Generated from apiextensions/v1/composition_environment.go by ../hack/duplicate_api_type.sh. DO NOT EDIT.
```

Comment:
how did this generated file get updated (or deleted actually) as part of this PR? i don't see other api source files being updated that would obviously lead to this - can you help me connect the dots? 😇